### PR TITLE
Options Menu

### PR DESCRIPTION
**Pull request by MiguelZh**

### CODE REVIEW

File: Drakhtar/Managers/SDLAudioManager.cpp [hunk 1]
```diff
@@ -141,3 +144,22 @@ void SDLAudioManager::fadeOutChannel(const int channel, const int ticks) {
...
 void SDLAudioManager::fadeOutMusic(const int ticks) { Mix_FadeOutMusic(ticks); }
+
+bool SDLAudioManager::getMuted() { return muted_; }
+
+void SDLAudioManager::setMute(bool mute) {
+  muted_ = mute;
+  checkMuted();
+}
+
+void SDLAudioManager::checkMuted() {
```
Comment (kyranet) [inline Drakhtar/Managers/SDLAudioManager.cpp]:
```cpp
const auto volume = muted_ ? 0 : 80;
const auto instance = getInstance();

instance->setChannelVolume(volume, 0);
instance->setChannelVolume(volume, 1);
instance->setMusicVolume(volume);
```

File: Drakhtar/Managers/SDLAudioManager.h [hunk 2]
```diff
@@ -47,9 +48,15 @@ class SDLAudioManager final : public AudioManager {
   void fadeOutChannel(int channel, int ticks);
   void fadeOutMusic(int ticks);
 
+  // check if its muted
+  bool getMuted();
+  void setMute(bool mute);
+  void checkMuted();
```
Comment (kyranet) [inline Drakhtar/Managers/SDLAudioManager.h]: This should be private, as it's only used internally.

File: images/UI/desktop.ini [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```
Comment (kyranet) [inline images/UI/desktop.ini]: ¿Qué es esto?

Comment (MiguelZh) [reply images/UI/desktop.ini]: Ni idea, salió solo metiendo un archivo

File: Drakhtar/Managers/SDLAudioManager.h (hunk 2 above)
Comment (MiguelZh) [reply Drakhtar/Managers/SDLAudioManager.h]: It is called in the OptionsMenu button when clicked, so it cannot be put in private

File: Drakhtar/Managers/SDLAudioManager.cpp (hunk 1 above)
Comment (MiguelZh) [reply Drakhtar/Managers/SDLAudioManager.cpp]: Done

**Pull request merged**